### DISCUSSION TITLE
Proxmox VE Alpine-Docker LXC - unable to mount SMB

### DISCUSSION THREAD

**Discussion by Vendo232** [tteck/Proxmox#1345] [Help Needed] [answered]
I have deployed Proxmox VE Alpine-Docker LXC and via console trying to mount SMB drive using this command

Proxmox VE Alpine-Docker LXC

`mount -t cifs -o username='login',password='pass*' //192.168.1.138/Public /media/movies`

and getting this msg
`mount error(1): Operation not permitted
Refer to the mount.cifs(8) manual page (e.g. man mount.cifs) and kernel log messages (dmesg)
mount: permission denied (are you root?)`

using command
`echo 0 > /proc/sys/kernel/dmesg_restrict`
returns
`-ash: can't create /proc/sys/kernel/dmesg_restrict: Read-only file system`

`alpine-docker:~# dmesg

dmesg: klogctl: Operation not permitted`

any idea what might be wrong, IP, credentials and new path is correct

**Comment by tteck** [comment]
Did you add cifs-utils? 
`apk add cifs-utils`

**Reply by Vendo232** [reply]
yes

let me try restart -> did not help

**Reply by tteck** [reply, marked as answer]
Is the LXC privileged?

**Reply by Vendo232** [reply]
that might be it, how do I check/change ?

**Comment by tteck** [comment]
By default, Alpine-Docker is created with unprivileged access unless specifically modified during the creation process using the advanced settings.

**Comment by Vendo232** [comment]
thank you tteck for your help, will have to research how to enter advanced setting

I would like to also change RAM and Storage during install rather after

got it

![image](https://user-images.githubusercontent.com/44098230/232848068-65150f76-3007-411f-ba7f-de6a0761828e.png)

**Reply by Vendo232** [reply]
this is awesome what you are doing tteck!
